packages/next/server/web/spec-extension/response.ts

```diff
@@ -35,36 +30,7 @@ export class NextResponse extends Response {
   }
 
   public get cookies() {
-    return this[INTERNALS].cookieParser()
-  }
-
-  public cookie(
-    name: string,
-    value: { [key: string]: any } | string,
-    opts: CookieSerializeOptions = {}
-  ) {
-    const val =
-      typeof value === 'object' ? 'j:' + JSON.stringify(value) : String(value)
-
-    const options = { ...opts }
-    if (options.maxAge) {
-      options.expires = new Date(Date.now() + options.maxAge)
-      options.maxAge /= 1000
-    }
-
-    if (options.path == null) {
-      options.path = '/'
-    }
-
-    this.headers.append(
-      'Set-Cookie',
-      cookie.serialize(name, String(val), options)
-    )
-    return this
-  }
-
-  public clearCookie(name: string, opts: CookieSerializeOptions = {}) {
-    return this.cookie(name, '', { expires: new Date(1), path: '/', ...opts })
```

Comment:
Note that clear a cookie after this PR is merged mean it will be not present at `set-cookie` header rather than force it to have an expired timestamp associated.